tests/utils/k8s.go

```diff
@@ -161,6 +162,59 @@ func waitForPodStatusPhase(clientSet *kubernetes.Clientset, podName string, name
 	})
 }
 
+func EnvVarInPod(envVar v1.EnvVar, pod v1.Pod) bool {
+	for _, e := range pod.Spec.Containers[0].Env {
+		if e.Name == envVar.Name && e.Value == envVar.Value {
+			log.Infof("Found %s=%s environment variable in first container of pod %s/%s", e.Name, e.Value, pod.Namespace, pod.Name)
+			return true
+		}
+	}
+	return false
+}
+
+/* ConfigMap */
+
+func CreateConfigMap(clientSet *kubernetes.Clientset, name string, namespace string) error {
+	log.Infof("Creating ConfigMap %s/%s", namespace, name)
+	_, err := clientSet.CoreV1().ConfigMaps(namespace).Create(&v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+	})
+	return err
+}
+
+func AddFileToConfigMap(clientSet *kubernetes.Clientset, cmName string, namespace string, key string, filepath string) error {
+	log.Infof("Adding %s to the ConfigMap %s/%s under key %s", filepath, namespace, cmName, key)
+	cm, err := clientSet.CoreV1().ConfigMaps(namespace).Get(cmName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	b, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return err
+	}
+
+	if cm.Data == nil {
+		cm.Data = make(map[string]string)
+	}
+
+	cm.Data[key] = string(b)
+	cm, err = clientSet.CoreV1().ConfigMaps(namespace).Update(cm)
+
+	return err
+}
+
+func DropConfigMap(clientSet *kubernetes.Clientset, name string, namespace string) error {
```

Comment:
As all the K8s API calls named `Delete*` why don't we name this function in a similar way like `DeleteConfigName` for the consistency purposes?